xbmc/pvr/epg/EpgContainer.cpp

```diff
@@ -118,7 +118,9 @@ CPVREpgContainer::CPVREpgContainer(void) :
   m_settings({
     CSettings::SETTING_EPG_IGNOREDBFORCLIENT,
     CSettings::SETTING_EPG_EPGUPDATE,
-    CSettings::SETTING_EPG_DAYSTODISPLAY
+    CSettings::SETTING_EPG_FUTURE_DAYSTODISPLAY,
+    CSettings::SETTING_EPG_PAST_DAYSTODISPLAY,
+    CSettings::SETTING_EPG_PREVENTUPDATESWHILEPLAYINGTV
```

Comment:
Where did this come from?

Reply:
+ SETTING_EPG_DAYSTODISPLAY renamed to CSettings::SETTING_EPG_FUTURE_DAYSTODISPLAY
+ SETTING_EPG_PAST_DAYSTODISPLAY newly introduced
+ SETTING_EPG_PREVENTUPDATESWHILEPLAYINGTV strictly speaking unrelated, but noticed that it was forgotten to keep this value in local settings cache (and to obtain it from there - another unrelated change in this PR - but consistent) ;-)